### ISSUE TITLE
Pydicom empty file error 

### ISSUE THREAD

**Issue by mckenziephagen** [nipy/heudiconv#488]
### Summary

Heudiconv failing when encountering an empty dicom file, due to a pydicom error.  [This error has since been solved ](https://github.com/pydicom/pydicom/issues/810) in newer versions of pydicom, but heudiconv is still using an older version. 

```
Traceback: 
INFO: Running heudiconv version 0.5.4
INFO: Need to process 1 study sessions
INFO: PROCESSING STARTS: {'subject': '29', 'outdir': '/oak/stanford/groups/russpold/data/network_grant/temp_raw_downloads/russpold/r01network/out/', 'session': '22439'}
INFO: Processing 353148 dicoms
INFO: Analyzing 353148 dicoms
/src/heudiconv/heudiconv/dicoms.py:58: UserWarning: The DICOM readers are highly experimental, unstable, and only work for Siemens time-series at the moment
Please use with caution.  We would be grateful for your help in improving them
  import nibabel.nicom.dicomwrappers as dw
Traceback (most recent call last):
  File "/opt/miniconda-latest/bin/heudiconv", line 11, in <module>
    load_entry_point('heudiconv', 'console_scripts', 'heudiconv')()
  File "/src/heudiconv/heudiconv/cli/run.py", line 127, in main
    process_args(args)
  File "/src/heudiconv/heudiconv/cli/run.py", line 323, in process_args
    dcmconfig=args.dcmconfig,)
  File "/src/heudiconv/heudiconv/convert.py", line 160, in prep_conversion
    grouping=None)
  File "/src/heudiconv/heudiconv/dicoms.py", line 61, in group_dicoms_into_seqinfos
    mw = dw.wrapper_from_data(dcm.read_file(filename, force=True))
  File "/opt/miniconda-latest/lib/python3.6/site-packages/pydicom/filereader.py", line 880, in dcmread
    force=force, specific_tags=specific_tags)
  File "/opt/miniconda-latest/lib/python3.6/site-packages/pydicom/filereader.py", line 693, in read_partial
    fileobj.seek(-1, 1)
OSError: [Errno 22] Invalid argument 
```

### Platform details:

Choose one:
- [ ] Local environment

- [x ] Container

Singularity container: heudiconv_0.5.4.sif
=

**Comment by pvelasco** [comment]
It looks like that the bug fix was rolled out for [pydicom 1.3.0](/pydicom/pydicom/commit/356a51ab4bc54fd18950041ebc44dbfa1a425a10).
The [Docker image](https://hub.docker.com/layers/nipy/heudiconv/0.9.0/images/sha256-42a4ab94a4af122e1031648f28dbc40a05b8b6cd6e720c7e94e539e93a519636) for the latest `heudiconv` release ([0.9.0](/nipy/heudiconv/releases/tag/v0.9.0)) seems to use pydicom 2.1.2, so it looks like it should work for your case.

However:
I was wondering why you were using version `0.5.4`, which is an old version. It looks like the problem is that the [official documentation](https://heudiconv.readthedocs.io/en/latest/installation.html#singularity) hasn't been updated, even though [`installation.rst`](blob/master/docs/installation.rst) is up to date.

**Comment by mckenziephagen** [comment]
Oh, I was using 0.5.4 because of I used the documentation to install - I'll rebuild and confirm that that gets rid of the problem. 

Thanks for pointing that out!
